en/components/data-chart-navigation.md

```diff
@@ -85,7 +85,7 @@ The following code snippet demonstrates how you can enable UI navigation in the
 
 <!-- ### Chart Navigation with Overview Plus Detail Pane
 
-In the `Ig$DataChart` control, there is an overlaid control that allows navigation. This control supports a preview of most supported series types as well as all navigation methods described above. This overlay is the overview plus detail pane, and it can be enabled by setting the `overviewPlusDetailPaneVisibility` property.
+In the `XamDataChart` control, there is an overlaid control that allows navigation. This control supports a preview of most supported series types as well as all navigation methods described above. This overlay is the overview plus detail pane, and it can be enabled by setting the `overviewPlusDetailPaneVisibility` property.
```

Comment:
Should this be `XamDataChart`?

Reply:
Yes, the translations are done from the C# type.

Reply:
this should not be XamDataChart by the time it reaches the angular output. I think something went wrong with the translation

Reply:
in xplat-docfx it should be XamDataChart, but by the time it gets here it should not.

Reply:
oh nm, we are in xplat docfx atm

Reply:
no, nm, I was right initially
we are not :)

Reply:
It's seeming very much like the translation broke and stopped mapping the type names. This shouldn't be merged like this and we need to investigate.

Reply:
This is ok, actually, it is inside a commented out area, so the translation is ignoring the code terms, accordingly.